deploy/pvc-cad.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: cad-pvc
```

Comment:
does that mean that only one run can happen in parralel ? (only one PVC can be bound to one pod

Reply:
Generally, mounting a PV to several pods shouldn't be an issue. But yes, we have to change the accessMode. 
As we discussed, the idea with the PV was to have a persistent way of storing logs for debugging. As execution information is lost, once the pod is deleted. But maybe there is a tektonish way to solve that without PVC. I am not sure right now.